xml/System.Globalization/CompareInfo.xml

```diff
@@ -3097,14 +3107,24 @@ Use <xref:System.Globalization.CompareInfo.GetSortKeyLength(System.ReadOnlySpan{
         <Parameter Name="matchLength" Type="System.Int32" RefType="out" Index="3" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="suffix">To be added.</param>
```

Comment:
I see the `suffix` -> `prefix` change you mentioned, @gewarren. Thanks for fixing it.